--- a/node_modules/react-native-pdf-renderer/android/src/main/java/com/github/douglasjunior/reactNativePdfRenderer/modules/PdfRendererRecyclerView.java
+++ b/node_modules/react-native-pdf-renderer/android/src/main/java/com/github/douglasjunior/reactNativePdfRenderer/modules/PdfRendererRecyclerView.java
@@ -298,7 +298,11 @@ public class PdfRendererRecyclerView extends RecyclerView {
             if (source == null || source.isEmpty()) return;
             File file = new File(source.replace("file://", ""));
             mFileDescriptor = ParcelFileDescriptor.open(file, ParcelFileDescriptor.MODE_READ_ONLY);
-            mPdfRenderer = new PdfRenderer(mFileDescriptor);
+            try {
+                mPdfRenderer = new PdfRenderer(mFileDescriptor);
+            } catch (IOException e) {
+                System.out.println(e.getMessage());
+            }
         }

         public void close() throws IOException {
